pkgs/development/python-modules/pandoc-eqnos/default.nix

```
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, fetchFromGitHub
+, lib
+, pandoc-xnos
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-eqnos";
+  version = "2.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "tomduck";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-7GQdfGHhtQs6LZK+ZyMmcPSkoFfBWmATTMejMiFcS7Y=";
+  };
+
+  propagatedBuildInputs = [ pandoc-xnos ];
+
```

The upstream repo contains [tests](https://github.com/tomduck/pandoc-eqnos/tree/master/test) but it looks like that they need to be generated on the fly.

```suggestion

  # Different pandoc executables are not available
  doCheck = false;

  pythonImportsCheck = [
    "pandoc_eqnos"
  ];
```